index.html

```diff
@@ -10,11 +10,13 @@
 
 $(document).ready(function(){
 
-    $('#button').click(function(e) {  
-        var inputvalue = $("#input").val();
-        var inputvalue2 = $("#input2").val();
-        window.location.replace(" http://www.paypal.me/"+inputvalue+"/"+inputvalue2);
+    $('#button').click(function(e) {
+      var username = $("#input").val();
+      var amount = $("#input2").val();
 
+      var token = window.btoa(username + ':' + amount).replace(/\=*$/m, '');
```

Comment:
Here's we're grabbing the input values once again, like you already had. Then we pass them to [`btoa`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa) the "encode" function. 

Due to how base64 works, if the string isn't of correct "length" then base64 will add "padding" which is one or two equals signs. This looks messy for us, and doesn't really mean anything: you can still decode without padding. See: https://en.wikipedia.org/wiki/Base64#Output_padding

So, we just remove those equals signs.